src/features/layout/Path.tsx

```diff
@@ -20,6 +26,7 @@ export default function Path() {
             data-testid='header-path'
             classes={classes}
             maxItems={4}
+            separator={<SeparatorIcon fontSize='small'></SeparatorIcon>}
         >
             {createPathPoints(['main', 'data'])}
```

Comment:
hardcoded value

Reply:
link is non-clickable (no route specified)

Reply:
please support a case when route and label might be different. for example `/data/my-custom-data-route` should be displayed as `Data -> Custom`

Reply:
you can fix Breadcrumbs in a separate branch (and with separate PR) later on 'cause it looks like a separate task